app/js/components/LabOrdersList.jsx

```diff
@@ -53,13 +53,13 @@ const Cell = ({ columnName, value, dateAndTimeFormat }) => {
     case 'ORDER DATE':
       return (
         <div className="table_cell order-date">
-          <span>{moment(value.dateActivated).format(dateAndTimeFormat || "DD-MMM-YYYY HH:mm")}</span>
+          <span>{moment(value.dateActivated).format(dateAndTimeFormat.split(' ')[0] || "DD-MMM-YYYY")}</span>
```

Comment:
We can't guarantee that we can do a "split" here, unfortunatel, you could have a date/time format like: "DD MMM YYYY HH:mm"?

Unfortunately, I think we need to add a new constant for just "dateFormat".

(Long term we can make these GPs supplied by the React Components module, so each module doesn't need to define these separately... I will ticket that)